tbclient/models.go

```diff
@@ -165,10 +183,11 @@ type VmNic struct {
 }
 
 type VmAdvancedSettings struct {
-	NameInHypervisor      string `json:"nameInHypervisor,omitempty"`
-	BiosUuid              string `json:"biosUuid,omitempty"`
-	NotStarted            bool   `json:"notStarted"`
-	AllDisksNonPersistent bool   `json:"allDisksNonPersistent"`
+	NameInHypervisor      string  `json:"nameInHypervisor,omitempty"`
+	BiosUuid              string  `json:"biosUuid,omitempty"`
+	NotStarted            bool    `json:"notStarted"`
+	AllDisksNonPersistent bool    `json:"allDisksNonPersistent"`
+	EvcMode               *string `json:"evcMode"`
```

Comment:
`string` defaults to `""` so if we go with `string` instead of `*string` and add `omitempty` then it'll behave the same as sending nil as the pointer when not set, and will make things a bit easier.

Reply:
(but test it, I'm 99% on it)